android/src/main/java/com/singular/flutter_sdk/SingularSDK.java

```diff
@@ -216,6 +217,15 @@ void initSDK() {
       singularConfig.withIMEI(imei);
     }
 
+    ArrayList<Map>  globalProps =  (ArrayList<Map>)configDict.get("globalProperties");
```

Comment:
I think there is few risks in this line we need to guard from. it is input from client so maybe casting will fail? what if the key is not existing? will the next line throw a null pointer exception?